test/unit/telemetry/telemetry-init.test.ts

```diff
@@ -0,0 +1,53 @@
+import { AcrolinxEndpoint, DEVELOPMENT_SIGNATURE } from 'src';
+import { AcrolinxInstrumentation, TelemetryConfig } from 'src/telemetry/acrolinxInstrumentation';
+import { ACROLINX_API_TOKEN, ACROLINX_DEV_SIGNATURE, TEST_SERVER_URL } from 'test/integration-server/constants';
+import { beforeEach, describe, expect, it } from 'vitest';
+import * as dotenv from 'dotenv';
+
+dotenv.config();
+
+describe('Telemtry initialization', () => {
+  let acrolinxEndpoint: AcrolinxEndpoint;
+  let props: TelemetryConfig;
+
+  beforeEach(() => {
+    acrolinxEndpoint = new AcrolinxEndpoint({
+      acrolinxUrl: TEST_SERVER_URL,
+      enableHttpLogging: true,
+      client: {
+        signature: ACROLINX_DEV_SIGNATURE ?? DEVELOPMENT_SIGNATURE,
+        version: '1.2.3.666',
+      },
+    });
+
+    props = {
+      accessToken: ACROLINX_API_TOKEN,
+      acrolinxUrl: acrolinxEndpoint.props.acrolinxUrl,
+      serviceName: 'sdk-js',
+      serviceVersion: '1.0.0',
```

Comment:
Note that in our logging (Grafana) we also have `"serviceName"` and `"serviceVersion"`. I'm wondering if this would cause any confusion using the same here, but with different values. 

For example here it is the integration name and version. In logs the fields `service_name` and `service_version` is our Integrations Service service name and the version.

Reply:
Open telemetry semantics are noted here: https://github.com/open-telemetry/opentelemetry-js/blob/main/semantic-conventions/src/stable_attributes.ts#L645

They have dot separator. Shouldn't conflict